ci/jobs/pulp-dev.yaml

```diff
@@ -23,6 +23,8 @@
         - shell:
             !include-raw-escape:
                 - 'ssh-setup.sh'
+        - shell:
```

Comment:
We could have everything on the same shell build step, but it is fine to leave this way.